language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -302,6 +302,9 @@ private List<LSCompletionItem> getTypeItems(BallerinaCompletionContext context)
                 String label = completionItem.getLabel();
                 String moduleName = importDeclarationNode.prefix().isEmpty() ? moduleSymbol.id().moduleName()
                         : importDeclarationNode.prefix().get().prefix().text();
+                if (moduleName.contains(".")) {
+                    moduleName = moduleName.split("\\.")[1];
+                }
```

Comment:
This will break for a scenario where we have more than one `import-identifier`. In that case we should select the later `import-identifier`s as well. WDYT?

Reply:
+1